engine-transactions/src/lib.rs

```diff
@@ -1,5 +1,9 @@
-use crate::prelude::types::{Address, Wei};
-use crate::prelude::{vec, TryFrom, Vec, H160, U256};
+#![cfg_attr(not(feature = "std"), no_std)]
+#![cfg_attr(not(feature = "std"), feature(alloc_error_handler))]
+#![cfg_attr(feature = "log", feature(panic_info_message))]
```

Comment:
Are these features needed?

Reply:
They shouldn't be, unless we are using something from std.

Reply:
We use `std` for tests. Without that, we can't run tests.